Web UI/CognitiveSearch.UI/Controllers/HomeController.cs

```diff
@@ -65,6 +65,23 @@ public IActionResult Search(string q)
 
             return View();
         }
+        public IActionResult Images()
+        {
+            return View();
+        }
+        [HttpPost, HttpGet]
+        public IActionResult Images(string q)
```

Comment:
What happens if the incoming string is empty? Could there be any bad side effects with injection or enforcing ownership/rights to a query?

Reply:
Oh no, this part of the code, just deals with logged the searched query term and searchId to application insights.
So, if this incoming string is empty, it should log a blank string as the query term